components/dependency_injection/compilation.rst

```diff
@@ -500,7 +500,8 @@ serves at dumping the compiled container::
     The ``file_put_contents()`` function is not atomic. That could cause issues
     in a production environment with multiple concurrent requests. Instead, use
     the :ref:`dumpFile() method <filesystem-dumpfile>` from Symfony Filesystem
-    component or other methods provided by Symfony (e.g. ``$containerConfigCache->write()``)
+    component or other methods provided by Symfony (e.g. ``$containerConfigCache->write()``
+    which is part of the :doc:`Config component </components/config>`)
     which are atomic.
```

Comment:
the atomic part relates to the method, not to the config component

Reply:
@OskarStark true, my point is just to mention where $containerConfigCache->write() may be found.
I think now it's correct, the part
```
  which are atomic.
```
goes after brackets and refers to component or other methods provided by Symfony